desktop/release-notes.md

```diff
@@ -24,12 +24,94 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.15.0
+
+2022-12-01
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Added webassembly capabilities, use with the beta containerd integration.
+- Containers will be restored on restart with the containerd integration.
+- Fixed listing multi-platform images with the containerd integration.
+- Better handling of dangling images with the containerd integration.
+- Implement "reference" filter for images with the containerd integration.
+- Add support for selecting upstream HTTP/HTTPS proxies automatically via `proxy.pac` in containers, `docker pull` etc.
+- Support custom names for new Dev Environments.
+- Updated the beta features settings to better describe experimental features and how people can access them.
+- Show available disk space of VM in footer.
+- Show disk space warning in footer if available space is below 3GB.
+- Support creating Dev Environments by cloning a Git repository to a local bind mount.
+- Updated colours and typography to be in line with Docker Design System.
+- Added a Build tab inside Extensions containing all necessary resources to build an extension.
+- Added the ability to share extensions more easily, either with `docker extension share` CLI or the share button in extensions "Manage" tab.
+- Display number of installs in the extensions marketplace, and ability to sort by number of installs.
+- Adds button to open DevEnv folder from UI.
```

Comment:
Is this meant to be about the browser extension @hyu ?

Reply:
No, this refers to a [change](https://github.com/docker/pinata/pull/19709) where the "open folder" button for Dev Envs works more consistently.

I mentioned the Browser Extension in the [blog](https://docs.google.com/document/d/1Qn-_xfGMl96UqGuh6szdv1ZTd3-NG74gjX0DBKtFjqo/edit#heading=h.nm1ak21mhyx1), but not in these release notes since it's not strictly a part of the DD 4.15 release.